classes/locations.php

```diff
@@ -293,7 +293,7 @@ function geo_location( $post ) {
 
 					<!-- Lat / Lng -->
 					<tr valign="top">
-						<td><label for="location_lat"><?php _e( 'Latitude/Longitude', 'simplemap' ); ?></label></td>
+						<td><label for="location_lat"><?php _e( 'Latitude/Longitude', 'SimpleMap' ); ?></label></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->